.gitignore

```diff
@@ -1,3 +1,9 @@
+# START_FEATURE django_react
```

Comment:
I'm fine having these be unconditional gitignore includes (node_modules should never get checked in to git; seems legit).  I.e. no feature flag needed here